apps/benchmark/tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
+  "extends": "../../tsconfig.base.json",
+  "compilerOptions": {
+    "allowJs": true,
```

Comment:
```suggestion
    "allowJs": true,
    "checkJs": true,
```

Reply:
Done in [6303a52](https://github.com/microsoft/griffel/pull/132/commits/6303a52d35a5ace09f5d3a05ddf9c4f0ff413b40)